run.go

```diff
@@ -177,8 +177,11 @@ func DefaultNamespaceOptions() NamespaceOptions {
 		{Name: string(specs.UserNamespace), Host: true},
 		{Name: string(specs.UTSNamespace), Host: true},
 	}
-	g := generate.New()
-	spec := g.Spec()
+	g, err := generate.New("linux")
+	if err != nil {
+		return options, err
+	}
+	spec := g.Config
```

Comment:
nit: This is fine, but you may want to consider renaming `spec` -> `config` as well.  I don't know why the runtime-spec authors decided to use `Spec` for the structure backing `config.json`, but that JSON is a configuration based on a specification, not a specification in its own right (also why the file is not `specification.json` ;).